src/GroupManager.php

```diff
@@ -110,6 +110,11 @@ public function removeGroup($entity_type_id, $bundle_id) {
 
       if ($search_key !== FALSE) {
         unset($groups[$entity_type_id][$search_key]);
+        // If the last group of this entity type is removed, also remove the
+        // empty container.
+        if (empty($groups[$entity_type_id])) {
```

Comment:
this seems a bit to delicate -- which code is being "fooled" by the empty container?